app/controllers/admin/action_pages_controller.rb

```diff
@@ -129,7 +129,9 @@ def events
     respond_to do |format|
       format.html do
         if @actionPage.enable_congress_message?
+          launch_date = Time.zone.parse("Nov 19 2019")
           action_events = @actionPage.events.where(name: 'Action')
+                                            .where("time > ?", launch_date)
```

Comment:
I think you could avoid the magic date here by only looking at events where the new property is present -- for example:

```ruby
where("json_extract_path(properties, 'customizedMessage') is not null")
```